.editorconfig

```diff
@@ -0,0 +1,123 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+###############################
+# Core EditorConfig Options   #
+###############################
+# All files
+[*]
+indent_style = space
+# Code files
+[*.{cs,csx,vb,vbx}]
+indent_size = 4
+insert_final_newline = true
+charset = utf-8-bom
+###############################
+# .NET Coding Conventions     #
+###############################
+[*.{cs,vb}]
+# Organize usings
+dotnet_sort_system_directives_first = true
```

Comment:
If I am not wrong, you are not sorting _using directives_ in this way.

Did you test if these editor config rules are really working?

Reply:
I think is better convention over configuration, for that, I want use the `.editorconfig` from template, and adapt current style and format

Thanks!

Reply:
Right, and I saw that you have applied the style change in the files. ❤️